android/AndroidSportsPlatform/app/src/main/java/com/example/sportsplatform/data/models/UserRegisterRequest.kt

```diff
@@ -3,10 +3,10 @@ package com.example.sportsplatform.data.models
 data class UserRegisterRequest(
         val email : String,
         val password : String,
-        val identifier : String,
-        val name : String,
-        val familyName : String,
-        val birthDate : String,
-        val gender : String,
-        val sports : String
+        val identifier : String
+       // val name : String,
```

Comment:
Later we should add these to our User Request.
Also we need to verify the entered password